codeowners

```diff
@@ -0,0 +1 @@
+* @maidsafe/backend_codeowners
```

Comment:
do we need to have a newline ?

Reply:
@maqi no, codeowners if 1 file that we don't need a newline